packages/wallets/src/features/cashier/flows/WalletDeposit/WalletDeposit.tsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
 import { useActiveWalletAccount } from '@deriv/api-v2';
-import { CashierLocked, DepositCryptoModule, DepositFiatModule, DepositLocked } from '../../modules';
+import { DepositCryptoModule, DepositFiatModule } from '../../modules';
 
 const WalletDeposit = () => {
     const { data } = useActiveWalletAccount();
     const isCrypto = data?.currency_config?.is_crypto;
 
-    return (
-        <CashierLocked module='deposit'>
-            <DepositLocked>{isCrypto ? <DepositCryptoModule /> : <DepositFiatModule />}</DepositLocked>
-        </CashierLocked>
-    );
+    return isCrypto ? <DepositCryptoModule /> : <DepositFiatModule />;
```

Comment:
Hi @heorhi-deriv,
I am curious why did we move DepositLocked out of WalletDeposit flow 😕.
Can we not just move CashierLocked out as that is more generic and has higher priority than the other locked scenarios?

Reply:
answered here: https://github.com/binary-com/deriv-app/pull/14585/files/0abef7d823f3fbd22ebcbb5919d7e95f04adf4df#:~:text=hey%20%40aum%2Dderiv,Deposit%2C%20Transfer%2C%20etc